lib/rules/multiple-assertions-checker.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
```

Comment:
Yeah, I thought it might not have been mentioned in the card, and yes to go to the effort of adding any descriptive text in your tests, and then make it the same in every case would be bizarre... 

I guess if it's a quick fix it might be worth adding, if not, well the rule is there to push you into doing the right thing, if you want to circumvent it there's always ways...